### PR TITLE
CN Version Setup guide _partials updates

### PR DESCRIPTION
**Pull request by zywangdylan**

### CODE REVIEW

**Comment by dmilon** [comment]
Hi @zywangdylan @The-Pavel :wave: 

As you could see [here](https://lewagon-alumni.slack.com/archives/GC05UC8MV/p1641495673040300), we released yesterday some setup updates. 

Can you propagate the changes in the Chinese version of the setup? Let me know if you have any question :pray:

**Comment by xunma** [comment]
Hi @dmilon , I updated all the partials according to #345. Please have a look and let me know if there's any feedbacks.  Thank you for reviewing!

Files modified are these:

- `_partials/cn/checkup.md`
- `_partials/cn/dotfiles.md`
- ` _partials/cn/git.md`
- `_partials/cn/macos_rbenv.md`
- `_partials/cn/nvm.md`
- `_partials/cn/rbenv.md`
- `_partials/cn/ruby.md`
- `_partials/cn/ubuntu_vscode.md`
- `_partials/cn/vscode_extensions.md`
- ` _partials/cn/windows_browser.md`
- `_partials/cn/windows_terminal.md`
- `_partials/cn/windows_ubuntu.md`
- `_partials/cn/windows_version.md`
- `_partials/cn/windows_wsl.md`
- `_partials/cn/zoom.md`

While I was translating, also fixed a small grammar error in this file:

-` _partials/windows_version.md`

File: _partials/windows_version.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Open Windows Update:
 - Press `Enter`
 - Click on `Check updates`
 
-:heavy_check_mark: If you a green check mark and the message "You're up to date", you're good to go :+1:
+:heavy_check_mark: If you see a green check mark and the message "You're up to date", you're good to go :+1:
```
Comment (dmilon) [inline _partials/windows_version.md:40]: Good catch :fishing_pole_and_fish: Thank you for the fix @xunma :pray:

**Review by dmilon — COMMENTED** [review]
Looking good to me :+1: Thank you for this work @xunma :pray: 

It seems that the [macos_dock.md Chinese partial](https://github.com/lewagon/setup/blob/21Oct-cn-setup-update/_partials/cn/macos_dock.md) is still here but doesn't have any corresponding English partial. Should we remove it?

**Comment by xunma** [comment]
hi @dmilon , thanks for the careful review~ I just removed the unused file~ Lemme know if there's anything else :)

**Pull request merged**